scripts/kafka-integration-test.sh

```diff
@@ -43,7 +44,16 @@ setup_kafka() {
   docker compose -f "${compose_file}" up -d kafka
 }
 
+dump_logs() {
+  echo "::group::Kafka logs"
+  docker compose -f "${compose_file}" logs
```

Comment:
please remove the corresponding step from the GH workflow (which is always no-op as it happens after docker compose shutdown)

Reply:
makes sense. my bad on the overlook there!